apps/web/www/components/header.tsx

```
@@ -87,44 +85,46 @@ const HeaderNavigationMenu = () => {
+                <>
+                  <NavigationMenuTrigger className="bg-transparent">
+                    {nl.name}
+                  </NavigationMenuTrigger>
+                  <NavigationMenuContent>
+                    <ul className="w-sm flex flex-col gap-2">
+                      {nl.children?.map((nl_x) => (
+                        <NavChild
+                          name={nl_x.name}
+                          description={nl_x.description}
+                          href={nl_x.link}
+                        />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)